README.md

```diff
@@ -302,16 +327,12 @@ Register a function by calling `watch`. For cases in which a callback is potenti
 ;; "watch function 2 fired. Fumbled? false"
 ```
 
-## Contributing
+## Maintainers and Contributors
 
-1. Branch and PR to master
-2. Maintainers will review.
+### Active Maintainers
 
-Guidelines:
+-
```

Comment:
Do we not have anyone to add to this list? :(

Reply:
No one explicitly yet, we can revisit and update these once everything is wrapped up.